### DISCUSSION TITLE
Medical balance changes.

### DISCUSSION THREAD

**Discussion by HeatedWafflez** [FakeFishGames/Barotrauma#4934] [Ideas and suggestions]
Currently the issue regarding the medical system is that it's fairly unrealistic, such as when morphine is a cure all. Although I have heard speculations on reworking the system entirely I know this won't happen in quite some time. An alternative proposition would be to instead balance what already exists in order to make things seem more immersive and realistic.

Anabolic Steroids, Methamphetamine, and Hyperzine were changed some point in the past to apply instantly. I'm not sure why this was added but it's incredibly overpowered, especially in combat missions where one of the combatants could inject themselves with either Hyperzine or steroids in order to heal themselves instantly and become quite literally unstoppable. I propose that this change be reverted so that the effects provided by these instead apply over time for the sake of immersion and so that players know that they need to anticipate situations where they need the extra strength.

Opiates, along with bandages and saline/blood packs work too quickly. I simply propose that these heal over a longer duration. I also suggest this because often when Fentanyl or Morphine is used by inexperienced players they find their patients overdosing. The issue regarding overdosing is that it happens far too quick and as a result it makes Naloxone, its cure, useless most of the time. This is also an issue because a full syringe gun of opiates can easily overdose somebody who won't be able to find the cure in time, let alone use it. 

When using a blood/saline pack you could have it apply over time as it heals blood loss. As it's being applied the durability on the blood pack will gradually go down as well. Any person regardless of their experience will drain the durability of the blood/saline pack at a constant rate with the main difference being that anybody with a high enough medical skill will apply it quicker.

Bandages taking time to apply, and the speed at which they are applied is influenced by the medical skill. The durability that is also taken from bandages is also dependent on the medical skill of those who are applying it. In the case of Antibiotic Glue it would work sort of like a blood/saline pack where it's applied over time and the speed at which it's applied is affected by medical skill.

An idea inspired by MrModeste was the changing of Delirumine. Currently it's regarded as a sort of toxin/poison with its own special cure. The issue with this is it simply applies psychosis that's identical to ordinary psychosis. My proposition is that Delirumine instead causes the player to lose control of their actions and behave aggressively towards everything. Sort of like paralysis you're unable to do anything but watch your psychotic character go on a rampage with what weapons they do have. This would be a nice gameplay aspect, especially for traitors trying to divert attention and frame others for what would be considered traitorous activity. Those who are afflicted would be at the mercy of their crew who may either try to save you or kill you.

The reality behind these suggestions is that they're just a foothold into what the medical system could be. Not all of these suggestions need to be applied, but they'll definitely help with balance, realism, and adding to the gameplay experience.

**Comment by Xeavas** [comment]
Current problem with changing the application of some treatments to be slower (bleeding and bloodloss) can't be done as people die **too fast**. So the med overhaul should take into consideration of prolonging death, and thus giving more time (and more available options that you can cram) into the patient.

**Comment by RadCatish** [comment]
Problem: no one uses anything but morphine. Why does morphine even cure half the things it does in Barotrauma, it's a painkiller?
Possible solutions: 
- Make opiates harder to come by - maybe harder to buy at stations, so EVA is still encouraged
- Make morphine way less effective (at least so you've got some incentive to use fentanyl/deusizine)
- Make the alternatives better - why would I use antibiotic glue which may cause organ failure which I'd need to cure with morphine, if I could just use morphine?

**Reply by RadCatish** [reply]
Oh, and I think that it should get more difficult to cure people as the difficulty increases/as  the campaign goes on. Maybe a morphine is enough to heal up from half health at <30% difficulty, but then you'll need a fentanyl for the same effects, and eventually a deusizine

**Reply by Kulik242** [reply]
I like the your previous ideas, but not thisone, the effect of meds should be constant throughout the campaign.
